orchestrate.py

```diff
@@ -226,7 +226,6 @@ def main():
     )
 
     spawner = dockworker.DockerSpawner(docker_host,
-                                       version=opts.docker_version,
```

Comment:
I think it's ok if they still want to specify it, since that's part of our current command line options. It's just the default that should be auto.